wizeline/wize-q-front
# Feat - WIP 251 - Remove name picture and update login message

### What does this PR do?

Removes name and picture from top of the page. It also adds a message to the login page in order to notify the users why login is required.

### Where should the reviewer start?

- Verify Login.js file. It now contains a message after the Login button.
- Verify App.js file. It no longer contains the username and profile picture at the top of the user interface.

### How should this be manually tested?

1. Go to login page and verify that message is shown without any typo or grammar errors.
2. Log in
3. Verify that your user name and profile picture no longer appear.

### Any background context you want to provide?

This aims to give some kind of certainty to the user that personal information is not being recorded.

### What are the relevant tickets?

[WIP-251 Remove Name, Picture and Logout button](https://wizeline.atlassian.net/browse/WIP-251)

### Screenshots

<img width="865" alt="screen shot 2017-06-22 at 2 27 14 pm" src="https://user-images.githubusercontent.com/2502032/27452552-89ef1920-5758-11e7-881c-b14775d66b82.png">
<img width="1213" alt="screen shot 2017-06-22 at 2 27 31 pm" src="https://user-images.githubusercontent.com/2502032/27452571-a4ca660a-5758-11e7-89cc-c4fd4cff80bf.png">
